.github/workflows/ci.yml

```
@@ -72,3 +72,21 @@ jobs:
         run: |
           PATH=$PATH:$PWD/tools/spell
           for lexfile in `find . -name lexicon.txt`; do dir=${lexfile%/lexicon.txt}; find-unknown-comment-words --directory $dir; if [ $? -ne "0" ]; then exit 1; fi; done;
+  fmt:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Uncrustify
+        run: sudo apt-get install uncrustify
+      - name: Run Uncrustify
+        run: find . -iname "*.c" -exec uncrustify --check -c tools/uncrustify.cfg {} +
```

This seems to be limited the check only on source files. What about the header files?
```suggestion
        run: find . -iname "*.[hc]" -exec uncrustify --check -c tools/uncrustify.cfg {} +
```